cmd/cni-metrics-helper/README.md

```diff
@@ -13,6 +13,9 @@ The following diagram shows how `cni-metrics-helper` works in a cluster:
 
 ![](../../docs/images/cni-metrics-helper.png)
 
+As you can see in the diagram, the `cni-metrics-helper` connects to the API Server
+over https (`tcp/443`), and another connection is created from the API Server to the worker node over http (tcp/61678). If you deploy Amazon EKS with recommended [Restricting cluster traffic](https://docs.aws.amazon.com/eks/latest/userguide/sec-group-reqs.html#security-group-restricting-cluster-traffic) then make sure that a security group is in place that allows the inbound connection from the API Server to the Worker Nodes over `tcp/61678`.
```

Comment:
nit: can you remove the newline after "API Server", add a comma before "then make sure", and change "Worker Nodes" to "worker nodes"?